### PR TITLE
Add /webapi/sites/:site/alerts endpoint to the apiserver

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Review by r0mant — COMMENTED** [review]
@hatched Can you add test coverage?

**Review by ibeckermayer — APPROVED** [review]
LGTM once you add the requested tests

File: lib/web/apiserver.go [hunk 1]
```diff
@@ -2111,6 +2114,34 @@ func (h *Handler) clusterNodesGet(w http.ResponseWriter, r *http.Request, p http
...
 
+type getLoginAlertsResponse struct {
+	Alerts []types.ClusterAlert `json:"alerts"`
+}
+
+// clusterLoginAlertsGet returns a list of on-login alerts for the user.
+func (h *Handler) clusterLoginAlertsGet(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+	// Get a client to the Auth Server with the logged in user's identity. The
+	// identity of the logged in user is used to fetch the list of alerts.
+	clt, err := ctx.GetUserClient(site)
```
Comment (zmb3) [inline lib/web/apiserver.go:2125]: Should this client be closed when we're done with it?

Comment (hatched) [reply lib/web/apiserver.go:2125]: It wasn't in any of the other endpoints but if you feel it's necessary I can add it.

**Comment by hatched** [comment]
@r0mant a test has been added, if you wouldn't mind taking another look.

File: lib/web/apiserver.go (hunk 1 above)
Comment (zmb3) [reply lib/web/apiserver.go:2129]: @Joerger what do you think? I'm not super familiar with these clients, but my guess is that it's probably an oversight and if there's a method that releases resources we should probably use it.

Comment (espadolini) [reply lib/web/apiserver.go:2129]: Auth clients are cached in the session and they're closed when the session gets cleaned up, AFAIK.

**Review by zmb3 — APPROVED** [review]

**Pull request merged**